xsd/netex_framework/netex_reusableComponents/netex_equipmentVehiclePassenger_support.xsd

```diff
@@ -111,12 +131,17 @@ Rail transport, Roads and Road transport
 		</xsd:annotation>
 		<xsd:restriction base="VehicleEquipmentIdType"/>
 	</xsd:simpleType>
-	<xsd:element name="WheelchairVehicleRef" type="InstalledEquipmentRefStructure" substitutionGroup="VehicleEquipmentRef">
+	<xsd:element name="WheelchairVehicleEquipmentRef" type="WheelchairVehicleEquipmentRefStructure" substitutionGroup="VehicleEquipmentRef">
 		<xsd:annotation>
 			<xsd:documentation>Reference to a WHEELCHAIR VEHICLE EQUIPMENT.</xsd:documentation>
 		</xsd:annotation>
 	</xsd:element>
-	<xsd:complexType name="WheelchairVehicleRefStructure" abstract="false">
```

Comment:
It is in master, but differently (different type). Needs to be discussed.
	<xsd:element name="WheelchairVehicleRef" type="InstalledEquipmentRefStructure" substitutionGroup="VehicleEquipmentRef">
		<xsd:annotation>
			<xsd:documentation>Reference to a WHEELCHAIR VEHICLE EQUIPMENT.</xsd:documentation>
		</xsd:annotation>
	</xsd:element>